packages/core/test/integration/http-handler.integration.ts

```diff
@@ -402,7 +410,14 @@ describe('HttpHandler', () => {
   }
 
   function givenClient() {
-    client = createClientForHandler(handler.handleRequest);
+    client = createClientForHandler((req, res) => {
+      handler.handleRequest(req, res).catch(err => {
+        console.error('Request failed.', err.stack);
```

Comment:
This catch block is needed to prevent unhandled rejection warnings in the test output.